internal/jimm/jimm.go

```diff
@@ -90,6 +90,9 @@ type JIMM struct {
 
 	// RoleManager provides a means to manage roles within JIMM.
 	RoleManager RoleManager
+
+	// RoleManager provides a means to manage groups within JIMM.
+	GroupManager GroupManager
```

Comment:
does the `GroupManager` need to be exported, or could we use the `GroupManager()` getter method instead?

Reply:
It doesn't need to be exported right now, but the plan is to remove the GroupManager() getter as soon as we can. Up to you whether you want me to make it exported or not right now.